packages/react-components/src/atoms/TextArea.tsx

```diff
@@ -113,11 +119,14 @@ const TextArea: React.FC<TextAreaProps> = ({
         </div>
         {maxLength !== undefined && (
           <div
-            css={[
-              validationMessageStyles,
-              limitStyles,
-              { color: reachedMaxLength ? ember.rgb : fern.rgb },
-            ]}
+            css={({ colors }) => {
```

Comment:
we can do something like `{colors:{primary500 = fern}={}}` here and just use primary500